assets/base.css

```
@@ -2751,11 +2752,17 @@ details[open] > .header__menu-item .icon-caret {
 
 .header__submenu .header__menu-item {
   justify-content: space-between;
-  padding: 0.8rem 2.4rem;
+  padding: 0.8rem 2rem;
 }
 
-.header__submenu .header__submenu .header__menu-item {
-  padding-left: 3.4rem;
+.header__submenu .header__menu-item:hover {
+  text-decoration: underline;
+  text-decoration-thickness: 0.2rem;
+}
+
+.header__submenu .header__submenu .header__menu-item:hover {
+  text-decoration: underline;
+  text-decoration-thickness: 0.2rem;
```

I think you only need one of those since they do the same thing. 
One has more specificity because you have an added `.header__submenu` but I don't think it's needed 👍 

Also I think the thickness is too big. We could remove that declaration 🤔 so it's thinner. That thickness should be used when it's an active link that already has an underline. I see that logic applied through this selector `.list-menu__item--active:hover` but it's being overwritten because `text-decoration` is a [shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration).

So I think if you just did this, it should work:
```suggestion
.header__submenu .header__menu-item:hover {
  text-decoration-line: underline;
}
```